src/universal/modules/teamDashboard/containers/Team/Team.js

```diff
@@ -0,0 +1,29 @@
+import React, {Component, PropTypes} from 'react';
+import Team from 'universal/modules/teamDashboard/components/Team/Team';
+import requireAuth from 'universal/decorators/requireAuth/requireAuth';
+import {connect} from 'react-redux';
+
+const mapStateToProps = (state, router) => {
```

Comment:
I feel the same way. I started righting a new top-level container and route and found that I was repeating myself a lot. Perhaps that's fine/better. I rewrite it tomorrow morning.